### PR TITLE
QACI-464 FISH-775 CmpRosterTest fails due to DirWatcher race dondition w/filesystem

### PR DESCRIPTION
**Pull request by lprimak**
added ASURLClassLoader.mustLoadFrom() to indicate that DirWatcher should be disregarded due to race conditions

## Description
There is an inherent race condition between OS delivering filesystem changes to DirWatcher,
when classes are loaded right after the class files are created.
Ignore DirWatcher when classes are necessary for app to work

## Testing
Fixes QuickLook stability
MacOS exposes this really well because it's a polling version of DirWatcher instead of pushing events

### CODE REVIEW

**Comment by lprimak** [comment]
jenkins test

**Comment by lprimak** [comment]
Jenkins test

**Comment by lprimak** [comment]
jenkins test

File: nucleus/common/common-util/src/main/java/com/sun/enterprise/loader/ASURLClassLoader.java [hunk 1]
```diff
@@ -120,6 +120,8 @@
     */
     private volatile String doneSnapshot;
 
+    private boolean mostLoadInThisStackFrame;
```
Comment (pdudits) [inline nucleus/common/common-util/src/main/java/com/sun/enterprise/loader/ASURLClassLoader.java:123]: Other than this typo, it looks good.

**Review by pdudits — APPROVED** [review]

File: nucleus/common/common-util/src/main/java/com/sun/enterprise/loader/ASURLClassLoader.java (hunk 1 above)
Comment (lprimak) [reply nucleus/common/common-util/src/main/java/com/sun/enterprise/loader/ASURLClassLoader.java:123]: ah, thanks!

**Comment by lprimak** [comment]
jenkins test

**Pull request merged**